crates/core/tedge_agent/src/error.rs

```diff
@@ -52,6 +52,15 @@ pub enum AgentError {
 
     #[error(transparent)]
     FromOperationsLogs(#[from] plugin_sm::operation_logs::OperationLogsError),
+
+    #[error(transparent)]
+    FromInfallible(#[from] std::convert::Infallible),
```

Comment:
This kind of contradicts what an `Infallible` is.

Reply:
This error case can simply be removed